.eslintrc

```diff
@@ -20,14 +20,17 @@
     "max-len": [
       "warn",
       {
-        "code": 150,
+        "code": 350,
```

Comment:
if we're going to put the char limit to anything bigger than 200 I think it might be better to take the rule out. this rule is to create better 'readability' of the code, which is important. I think it's better for us to use the inline comments to disable max-len on specific lines that we warrant are readable but long. (/* eslint-disable-next-line max-len */)